src/main/modules/helmet/index.ts

```diff
@@ -1,50 +1,52 @@
-import * as express from "express";
+import * as express from 'express';
 import helmet = require('helmet');
 
-export interface IConfig {
+export interface HelmetConfig {
   referrerPolicy: string;
 }
 
-const googleAnalyticsDomain = "*.google-analytics.com";
-const self = "'self'";
+const googleAnalyticsDomain = '*.google-analytics.com';
+const self = '\'self\'';
```

Comment:
the lint is wrong here, it should be the other way round
double quotes allowed when single quotes inside.

on cmc it worked I think,

would be good to support this